host/testdata/matching_simulation_default.yaml

```diff
@@ -24,5 +24,10 @@ matchingconfig:
       - taskprocesstime: 1ms
         numpollers: 8
         polltimeout: 60s
+    backlogs:
+      - partition: 1
+        backlogcount: 1000
+      - partition: 2
+        backlogcount: 2000
```

Comment:
Let's not modify `default` like this without changing all the other ones that derives from it.

Reply:
I recommend leaving default as is and create 2 new scenarios
- backlogged
- backlogged_weighted_lb

Reply:
I forgot to revert the change I used for local test.